tests/phpunit/tests/actions/closures.php

```diff
@@ -9,6 +9,9 @@ class Tests_Actions_Closures extends WP_UnitTestCase {
 
 	/**
 	 * @ticket 10493
+	 *
+	 * @covers ::has_action
+	 * @covers ::do_action
```

Comment:
This test is testing multiple things:

- `add_action`: registering a closure as the callback
- `has_action`: checking the closure was successfully registered as the callback
- `do_action`: closure was successfully invoked when the action event fired

It does cover all of these currently.

```suggestion
	 * @covers ::add_action
	 * @covers ::has_action
	 * @covers ::do_action
```

Note: This test is doing too much and should be split into separate tests (in another ticket).